### PR TITLE
Handle UserUpdated and UserDeleted events

### PR DESCRIPTION
**Pull request by doomspork**
This PR includes 3 changes:

1. Uses `ZenEx.Model.User.create_or_update/1` instead of `create/1`
2. Handles the `UserUpdated` message
3. Handles the `UserDeleted` message

### CODE REVIEW

File: lib/help_desk/broadway.ex [hunk 1]
```diff
@@ -66,6 +66,18 @@ defmodule HelpDesk.Broadway do
     notify_configured_handler(:users, :sync, message)
   end
 
+  defp notify_handler({:user_updated, message}) do
+    logger.metadata(user_id: message.user.id)
```
Comment (btkostner) [inline lib/help_desk/broadway.ex]:
```suggestion
    Logger.metadata(user_id: message.user.id)
```

File: lib/help_desk/broadway.ex [hunk 2]
```diff
@@ -66,6 +66,18 @@ defmodule HelpDesk.Broadway do
...
+    Logger.debug("Handling User Updated message")
+    notify_configured_handler(:users, :sync, message)
+  end
+
+  defp notify_handler({:user_deleted, message}) do
+    logger.metadata(user_id: message.user.id)
```
Comment (btkostner) [inline lib/help_desk/broadway.ex]:
```suggestion
    Logger.metadata(user_id: message.user.id)
```

**Review by btkostner — CHANGES_REQUESTED** [review]
Just those two little changes and everything else looks good

**Review by btkostner — APPROVED** [review]

File: lib/help_desk/users.ex [hunk 3]
```diff
@@ -5,9 +5,19 @@ defmodule HelpDesk.Users do
   alias Bottle.Account.V1.{User, UserCreated}
```
Comment (doomspork) [inline lib/help_desk/users.ex]: Need to swap out UserCreated for UserDeleted here 👍

**Pull request merged**